unit_tests/test_batch.py

```diff
@@ -91,32 +92,33 @@ def test_constructor_with_non_positive_batch_size(self):
 
     def test_constructor_with_batch_size_and_transactional(self):
         table = object()
+
         batch_size = 1
         transaction = True
         with self.assertRaises(TypeError):
             self._make_one(
                 table, batch_size=batch_size, transaction=transaction)
 
     def test_send(self):
-        table = object()
+        low_level_table = _MockLowLevelTable()
+        table = _MockTable(low_level_table)
+
         batch = self._make_one(table)
 
         batch._row_map = row_map = _MockRowMap()
-        row_map['row-key1'] = row1 = _MockRow()
-        row_map['row-key2'] = row2 = _MockRow()
+        row_map['row-key1'] = _MockRow()
+        row_map['row-key2'] = _MockRow()
         batch._mutation_count = 1337
 
         self.assertEqual(row_map.clear_count, 0)
-        self.assertEqual(row1.commits, 0)
-        self.assertEqual(row2.commits, 0)
         self.assertNotEqual(batch._mutation_count, 0)
         self.assertNotEqual(row_map, {})
+        self.assertEqual(len(table._low_level_table.rows_mutate), 0)
 
         batch.send()
         self.assertEqual(row_map.clear_count, 1)
-        self.assertEqual(row1.commits, 1)
-        self.assertEqual(row2.commits, 1)
         self.assertEqual(batch._mutation_count, 0)
+        self.assertEqual(len(table._low_level_table.rows_mutate), 2)
```

Comment:
Needs an assertion that the contents of `rows_mutate` match expected values.

Reply:
Added assertion to check expected values. Thanks.